app/src/main/java/org/mifos/mobilebanking/ui/fragments/LocationsFragment.java

```diff
@@ -67,22 +67,28 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onMapReady(GoogleMap googleMap) {
 
-        addMarker(googleMap);
+        LatLng headquarterLatLng = new LatLng(47.61115, -122.34481);
+
+        addMarker(googleMap, headquarterLatLng);
+        addAnimationToHeadquarter(googleMap, headquarterLatLng);
 
     }
 
-    private void addMarker(GoogleMap googleMap) {
-        for (LatLng latLng: getLatLngList()) {
-            googleMap.addMarker(new MarkerOptions().position(latLng));
-        }
+    private void addMarker(GoogleMap googleMap, LatLng headquarterLatLng) {
+
+        googleMap.addMarker(new MarkerOptions().position(headquarterLatLng)
+                .title(getString(R.string.mifos_initiative)));
+
+    }
+
+    private void addAnimationToHeadquarter(GoogleMap googleMap, LatLng headquarterLatLng) {
+
+        googleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(headquarterLatLng, 16.0f));
     }
 
     private List<LatLng> getLatLngList() {
```

Comment:
@miPlodder , as this function is not used anywhere so can you remove this.

Reply:
@dilpreet96  Will remove it and update the PR.